RELEASING.md

```diff
@@ -0,0 +1,38 @@
+* Choose the right version number:
+  * `x.0.0` is reserved for incompatible changes and require a milestone cycle.
+  * `x.y.0` is reserved for compatible changes.
+  * `x.y.z` is reserved for bugfixes that don't change the public API,
+* Tag the release:
+  * The tag must be called `vx.y.z`, e.g. `v3.0.0`.
+  * `git tag -a vx.y.z -m "vx.y.z"`
```

Comment:
```
git tag vx.y.z
```

works anyway, and I've never understood the practical difference with annotated tag.

Reply:
I always include `-a` but have no idea why 🤷‍♂️